com.chartboost.mediation.demo/Assets/Demo/Demo.cs

```diff
@@ -242,48 +235,106 @@ public void OnCreateBannerClick()
+        Log(result.Error == null ? "Successfully loaded banner" : result.Error?.Message);
+
+        _bannerAd.ResizeToFit = true;
     }
 
-    public void OnRemoveBannerClick()
+    public void OnHorizontalAlignmentChange()
     {
-        _bannerAd?.Destroy();
-        _bannerAd = null;
-        Log("Banner Removed");
+        _bannerAd.HorizontalAlignment = (ChartboostMediationBannerHorizontalAlignment) horizontalAlignmentDropdown.value;
     }
 
-    public void OnClearBannerClick()
+    public void OnVerticalAlignmentChange()
     {
-        if (_bannerAd == null)
-        {
-            Log("banner ad does not exist");
-            return;
-        }
-        _bannerAd.ClearLoaded();
-        Log("banner ad has been cleared");
+        _bannerAd.VerticalAlignment = (ChartboostMediationBannerVerticalAlignment)verticalAlignmentDropdown.value;
     }
 
     public void OnToggleBannerVisibilityClick()
     {
         if (_bannerAd != null)
         {
             _bannerAdIsVisible = !_bannerAdIsVisible;
-            _bannerAd.SetVisibility(_bannerAdIsVisible);
+            _bannerAd.gameObject.SetActive(_bannerAdIsVisible);
         }
         Log("Banner Visibility Toggled");
     }
+    
+    public void OnToggleBannerDraggabilityClick()
+    {
+        if (_bannerAd != null)
+        {
+            _bannerAdIsDraggable = !_bannerAdIsDraggable;
+            _bannerAd.Draggable = _bannerAdIsDraggable;
+        }
+        Log("Banner Draggability Toggled");
+    }
+    
+    public void OnResetBannerClick()
+    {
+        if (_bannerAd == null)
+        {
+            Log("banner ad does not exist");
+            return;
+        }
+        _bannerAd.ResetAd();
+        Log("banner ad has been reset");
+    }
 
-    private void DidLoadBanner(string placementName, string loadId, BidInfo info, string error)
+    public void OnRemoveBannerClick()
     {
-        _bannerAdIsVisible = true;
-        Log($"DidLoadBanner{placementName}: \nLoadId: ${loadId} \nPrice: ${info.Price:F4} \nAuction Id: {info.AuctionId} \nPartner Id: {info.PartnerId} \nError: {error}");
+        if (_bannerAd == null)
+        {
+            Log("banner ad does not exist");
+            return;
+        }
+
+        Destroy(_bannerAd.gameObject);
+        Log("banner ad has been destroyed");
+
     }
 
-    private void DidClickBanner(string placementName, string error) 
-        => Log($"DidClickBanner {placementName}: {error}");
+    private void WillAppearBanner()
+    {
+        Log($"WillAppearBanner {_bannerAd}");
+    }
+    
+    private void DidRecordImpressionBanner()
+    {
+        Log($"DidRecordImpressionBanner {_bannerAd}");
+    }
 
-    private void DidRecordImpressionBanner(string placementName, string error) 
-        => Log($"DidRecordImpressionBanner {placementName}: {error}");
+    private void DidClickBanner()
+    {
+        Log($"DidClickBanner {_bannerAd}");
+    }
+    
+    private void DidDragBanner(float x, float y)
+    {
+        
```

Comment:
Was this going to be logged as well? Or is there a diff usage for this?

Reply:
I don't think this should be logged.